City-of-Helsinki/helsinki-paatokset
# PP-147: Policymaker page

To test:
- `make fresh`
- View a policymaker (eg. https://helsinki-paatokset.docker.so/fi/paattajat/kaupunginvaltuusto)
- Edit the policymaker: fill out fieds for `policymaker_image' and `summary`  to view what the page looks like with these
- Try adding custom links using the `custom_navigation_links` field. These should be displayed in the navigation embedded to first sidebar
- Try linking tpr_unit -entities using the `contacts` field
- Verify that changes to page.html.twig has not broken any other pages

To correctly display all parts of the site, you should have fresh data from the migrations.
